code/ui/manager/src/components/preview/preview.tsx

```diff
@@ -85,11 +84,12 @@ const createCanvas = (id: string, baseUrl = 'iframe.html', withLoader = true): A
             }
           }, []);
 
+          const isBooting = !storySpecified && !storiesFailed;
```

Comment:
Could you add a little comment here? People might not understand what booting means